website/docs/r/emr_cluster.html.markdown

```diff
@@ -0,0 +1,313 @@
+    }
+
+    host_group {
+        host_group_name = "task_group"
+        host_group_type = "TASK"
+        node_count = "2"
+        instance_type = "ecs.g5.xlarge"
+        disk_type = "CLOUD_SSD"
+        disk_capacity = "80"
+        disk_count = "4"
+        sys_disk_type = "CLOUD_SSD"
+        sys_disk_capacity = "80"
+        instance_list = "[\"instance_id1\", \"instance_id2\"]"
+    }
+
+    high_availability_enable = true
+    option_software_list = ["HBASE","PRESTO",]
+    zone_id = "cn-huhehaote-a"
+    security_group_id = var.security_group_id == "" ? alicloud_security_group.default[0].id : var.security_group_id
+    is_open_public_ip = true
+    charge_type = "PostPaid"
+    vswitch_id = var.vswitch_id == "" ? alicloud_vswitch.vswitch[0].id : var.vswitch_id
+    user_defined_emr_ecs_role = "EMRUserDefineRole-Role1"
+    ssh_enable = true
+    master_pwd = "ABCtest1234!"
+}
+```
+
+
+## Argument Reference
+The following arguments are supported:
+
+* `name` - (Required) The name of emr cluster.
+* `emr_ver` - (Required, ForceNew) EMR Version, e.g. EMR-3.22.0. You can find the all valid EMR Version in emr web console.
+* `cluster_type` - (Required, ForceNew) EMR Cluster Type, e.g. HADOOP, KAFKA, DRUID, etc. You can find all valid EMR cluster type in emr web console.
+* `charge_type` - (Required, ForceNew) Charge Type for this cluster. Supported value: PostPaid or PrePaid. Default value: PostPaid.
+* `zone_id` - (Required, ForceNew) Zone ID, e.g. cn-huhehaote-a
+* `security_group_id` (Optional, ForceNew) Security Group ID for Cluster, you can also specify this key for each host group.
+* `vswitch_id` (Optional, ForceNew) Global vswitch id, you can also specify it in host group.
+* `option_software_list` (Optional, ForceNew) Optional software list.
+* `high_availability_enable` (Optional, ForceNew) High Available for HDFS and YARN. If this is set true, MASTER group must have two nodes.
+* `use_local_metadb` (Optional, ForceNew) Use local metadb. Default is false.
+* `ssh_enable` (Optional, ForceNew) If this is set true, we can ssh into cluster. Default value is false.
+* `master_pwd` (Optional, ForceNew) Master ssh password.
+* `eas_enable` (Optional, ForceNew) High security cluster (true) or not. Default value is false.
+* `user_defined_emr_ecs_role` (Optional, ForceNew) Alicloud EMR uses roles to perform actions on your behalf when provisioning cluster resources, running applications, dynamically scaling resources. EMR uses the following roles when interacting with other Alicloud services. Default value is AliyunEmrEcsDefaultRole.
+* `key_pair_name` (Optional, ForceNew) Ssh key pair.
+* `deposit_type` (Optional, ForceNew) Cluster deposit type, HALF_MANAGED or FULL_MANAGED.
+* `related_cluster_id` (Optional, ForceNew) This specify the related cluster id, if this cluster is a Gateway.
+* `host_group` - (Optional) Groups of Host, You can specify MASTER as a group, CORE as a group (just like the above example).
```

Comment:
Please use a block to show the element of `host_group`, like https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/website/docs/r/oss_bucket.html.markdown#block-cors_rule